pkg/kv/kvserver/batcheval/cmd_delete_range.go

```diff
@@ -47,6 +48,24 @@ func DeleteRange(
 	h := cArgs.Header
 	reply := resp.(*roachpb.DeleteRangeResponse)
 
+	// Use experimental MVCC range tombstone if requested. The caller is expected
+	// to have checked storage.CanUseExperimentalMVCCRangeTombstone() first.
+	if args.UseExperimentalRangeTombstone {
+		if args.Inline {
+			return result.Result{}, errors.AssertionFailedf("Inline can't be used with range tombstones")
+		}
+		if args.ReturnKeys {
+			return result.Result{}, errors.AssertionFailedf(
+				"ReturnKeys can't be used with range tombstones")
+		}
+		maxIntents := storage.MaxIntentsPerWriteIntentError.Get(&cArgs.EvalCtx.ClusterSettings().SV)
+		err := storage.ExperimentalMVCCDeleteRangeUsingTombstone(
+			ctx, readWriter, cArgs.Stats, args.Key, args.EndKey, h.Timestamp, maxIntents)
+		if err != nil {
+			return result.Result{}, err
+		}
+	}
```

Comment:
Do we fall through to MVCCDeleteRange here to get deletedKeys? Not obvious how those interact with eachother in that case. I think it deserves some at least brief clarification unless it is unintended and we want to return here.

Reply:
Oops, nice catch -- we shouldn't fall through here. Fixed.